	- add the viprinet ISDN USB device ID
	- some debug output
--- mISDN-1_1_9_2.orig/drivers/isdn/hardware/mISDN/hfcs_usb.c	2009-01-22 14:36:34.000000000 +0100
+++ mISDN-1_1_9_2/drivers/isdn/hardware/mISDN/hfcs_usb.c	2011-07-20 15:00:25.362001559 +0200
@@ -208,6 +208,12 @@ static struct usb_device_id hfcsusb_idta
 	                  {LED_SCHEME1, {0x88, -64, -32, -16},
 	                   "ZyXEL OMNI.NET USB II"}),
 	},
+	{
+	 USB_DEVICE(0x18d8, 0x1020),
+	 .driver_info = (unsigned long) &((hfcsusb_vdata)
+			  {LED_SCHEME1, {0, 1, 0, 0},
+			   "Viprinet ISDN"}),
+	},
 	{ }
 };
 
